transformers/events/new_cdp/converter_test.go

```diff
@@ -17,28 +17,34 @@
 package new_cdp_test
 
 import (
-	"github.com/makerdao/vulcanizedb/pkg/core"
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
 	"github.com/makerdao/vdb-mcd-transformers/transformers/events/new_cdp"
 	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
 	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 )
 
 var _ = Describe("NewCdp Converter", func() {
-	var converter = new_cdp.NewCdpConverter{}
+	var (
+		converter = new_cdp.Converter{}
+		db        = test_config.NewTestDB(test_config.NewTestNode())
```

Comment:
nbd but I've been thinking we can probably just skip this altogether and pass `nil` for the `db` if it's unused